Source/WebCore/loader/cache/CachedFont.cpp

```diff
@@ -64,10 +68,47 @@ void CachedFont::didAddClient(CachedResourceClient& client)
         downcast<CachedFontClient>(client).fontLoaded(*this);
 }
 
+static AtomString hashForFontData(const uint8_t* data, size_t size)
+{
+    auto cryptoDigest = PAL::CryptoDigest::create(PAL::CryptoDigest::Algorithm::SHA_256);
+    cryptoDigest->addBytes(data, size);
+    auto digest = cryptoDigest->computeHash();
+    return AtomString(makeString(base64Encoded(digest.data(), digest.size())));
```

Comment:
Would `makeAtomString()` work instead of `AtomString(makeString())`. This would likely be more efficient if it did.